tests/integrate/204_NO_KP_AFM/INPUT

```diff
@@ -8,7 +8,7 @@ orbital_dir	../../PP_ORB
 gamma_only      0
 
 #Parameters (2.Iteration)
-ecutwfc			50
+ecutwfc			20
```

Comment:
Please be care of the convergent magnetizations of 204*/304* should be non-zero.

Reply:
> Please be care of the convergent magnetizations of 204*/304* should be non-zero.

I see. However, it seems that the original setting cannot get non-zero magnetic moments. Anyway, I have solved the problem.